Include/cpython/code.h

```diff
@@ -138,6 +156,7 @@ typedef struct {
        Type is a void* to keep the format private in codeobject.c to force     \
        people to go through the proper APIs. */                                \
     void *co_extra;                                                            \
+    _PyCode_DEF_THREAD_LOCAL_BYTECODE()                                        \
```

Comment:
That is possible, but it would mean that we cannot fetch the tlbc array from a fixed offset. The code object is laid out such that there is a fixed-length part (what we see in `_PyCode_DEF`) followed by the variable-sized bytecode. `co_code_adaptive` is a flexible array member that points to the start of the variable-sized bytecode. In order to not change the struct layout we would need to put the tlbc array pointer after the variable-sized bytecode. Fetching it would require dynamically computing its location. I'd prefer not to do that, and instead have it live at a fixed offset, because we need to fetch it on every function call.

Reply:
Ah, sorry, I hadn't noticed `SIZE` was parameterised, sorry! Yes, that make sense.